vstorage/vstorage.go

```diff
@@ -0,0 +1,78 @@
+package vstorage
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+type Vstorage struct {
+	Name string
+}
+
+type Mntent struct {
+	Device string
+	Path   string
+	Type   string
+}
+
+func readMounts(path string) ([]Mntent, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	out := []Mntent{}
+	r := bufio.NewReader(f)
+	for {
+		line, err := r.ReadString('\n')
+		if err == io.EOF {
+			break
+		}
+		fields := strings.Fields(line)
+		if len(fields) < 3 {
+			continue
+		}
+
+		out = append(out, Mntent{
+			Device: fields[0],
+			Path:   fields[1],
+			Type:   fields[2],
+		})
+	}
+	return out, nil
+}
+func (v *Vstorage) Mountpoint() (string, error) {
```

Comment:
More thoughts on the topic - this way we will have 2nd mount of the same storage on vz7 nodes. I.e. we will not detect if it is already mounted on the node in some other place (like /vstorage, /pcs, etc.)